scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -63,7 +55,7 @@
                         "type": "Microsoft.Insights/dataCollectionRules",
                         "apiVersion": "2021-04-01",
                         "name": "[variables('dcrName')]",
-                        "location": "[parameters('workspaceLocation')]",
+                        "location": "[parameters('clusterRegion')]",
```

Comment:
Location of the DCR MUST be the same workspace location. DCR cant created with different region than workspace.

Reply:
yes, updated!